Makefile

```diff
@@ -55,6 +55,7 @@ clean:
 build/llvm.BUILT:
 	mkdir -p build/llvm
 	cd build/llvm && cmake -G Ninja \
+		-DCMAKE_MODULE_PATH=$(ROOT_DIR)/cmake \
```

Comment:
There is no cross building happening here so is this needed?

Reply:
Fixed.